test/modules/MultiToken.t.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "forge-std/Test.sol";
+
+import {Hooks} from "src/libraries/Hooks.sol";
+import {MultiTokenModule} from "src/modules/MultiToken.sol";
+
+import {Deployers} from "test/utils/Deployers.sol";
+
+contract MultiTokenModuleTest is Test, Deployers {
+    MultiTokenModule module;
+
+    /*//////////////////////////////////////////////////////////////
+                                 SETUP
+    //////////////////////////////////////////////////////////////*/
+
+    function setUp() public virtual {
+        deployGovernor(address(0));
+
+        module = MultiTokenModule(address(uint160(Hooks.BEFORE_VOTE_FLAG)));
+        deployCodeTo("src/modules/MultiToken.sol:MultiTokenModule", abi.encode(address(governor)), address(module));
+    }
+
+    function test_addToken() public {
+        module.addToken(address(token), 100, bytes4(keccak256("transfer(address,uint256)")));
```

Comment:
wouldn't we want to call `getVotes(address delegate)` and then parse the result from the proposal snapshot?

Reply:
I get that it's just a example here for testing but it gives the impression that we'd accept anything with this sig format